src/commands/import-resolver.ts

```diff
@@ -1,9 +1,9 @@
+import {Stats, statSync} from 'fs';
 import path from 'path';
 import {SnowpackConfig} from '../config';
 import {findMatchingMountScript} from '../util';
-import {isDirectoryImport} from './build-util';
 import srcFileExtensionMapping from './src-file-extension-mapping';
-
+const cwd = process.cwd();
```

Comment:
how do you feel about adding `cwd` to the config object? We do this in a few different places around the codebase, but it would be good to start moving towards a configurable cwd.

Reply:
I think that’s a great idea.